server/plugin.go

```diff
@@ -337,12 +343,16 @@ func (p *Plugin) handleRemove(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	issue, foreignID, isSender, err := p.listManager.RemoveIssue(userID, removeRequest.ID)
+	issue, foreignID, isSender, listToUpdate, err := p.listManager.RemoveIssue(userID, removeRequest.ID)
 	if err != nil {
 		p.API.LogError("Unable to remove issue, err=" + err.Error())
 		p.handleErrorWithCode(w, http.StatusInternalServerError, "Unable to remove issue", err)
 		return
 	}
+	p.sendRefreshEvent(userID, []string{listToUpdate})
+	if isSender {
+		p.sendRefreshEvent(userID, []string{OutListKey})
+	}
```

Comment:
Is this not redundant? If we are the sender, the list to update for the user is already the "Out" list, right?

Reply:
no, not really. In the current implementation the listToUpdate cannot be the "Out" list. So if the ToDo is also in the "Out" list, it'll be updated as well. Basically sometimes I need to update two lists.

Reply:
Why it cannot be out? Also, doublechecking the code, I just realized that "isSender" is referring to the foreigner, not the local user.

Reply:
It cannot be out, because it can happen that `RemoveIssue` function updates two lists at once. (In or My and optionally Out). I mean I could make `RemoveIssue` return an array of `listsToUpdate`. That would probably be best.